dietpi/dietpi-software

```diff
@@ -10720,11 +10723,15 @@ _EOF_
 [Unit]
 Description=DietPi Gogs service
 DefaultDependencies=no
+After=mysqld.service mysql.service mariadb.service
```

Comment:
In DietPi, these are not required. As ```dietpi-services``` controls these services and has its own starting/stopping ordering system.

However, there is no harm in leaving these in :)

Reply:
@Fourdee 
Jep. as mentioned this is just fail safe anyway 🙂.